Api/MenuRepositoryInterface.php

```diff
@@ -6,15 +6,43 @@
 
 interface MenuRepositoryInterface 
 {
+    /**
+     * @param \Snowdog\Menu\Api\Data\MenuInterface $page
+     * @return \Snowdog\Menu\Api\Data\MenuInterface
+     */
     public function save(MenuInterface $page);
 
+    /**
+     * @param int $id
+     * @return \Snowdog\Menu\Model\Menu
+     */
     public function getById($id);
 
+    /**
+     * Returns menus list
+     *
+     * @api
+     * @param \Magento\Framework\Api\SearchCriteriaInterface $criteria
+     * @return \Snowdog\Menu\Api\Data\MenuSearchResultsInterface|string
```

Comment:
I think we should update the return object (type) in our implementation of this interface too: https://github.com/SnowdogApps/magento2-menu/blob/master/Model/MenuRepository.php#L66

Reply:
Updated